File: 2310817120009_AlysaArmelia_PemrogramanI_Modul6/Python/PRAK602-2310817120009-AlysaArmelia.py
zetsu_putih = int(input())

arr = list(map(int, input().split()))

ruangan = 0

while(ruangan < zetsu_putih):
    hasil_akhir = arr[ruangan] * (ruangan +1)
    print (hasil_akhir, end=" ")
    ruangan += 1

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul6/Bahasa C/PRAK602-2310817120009-AlysaArmelia.c
#include <stdio.h>

int main()
{
    int zetsu_putih;

    scanf("%d", &zetsu_putih);

    int arr[zetsu_putih];
    int ruangan;

    for (ruangan = 0; ruangan < zetsu_putih; ruangan++)
    {
        scanf("%d", &arr[ruangan]);
    }
    
    for (ruangan = 0; ruangan < zetsu_putih; ruangan++)
    {
        int hasil_akhir = arr[ruangan] * (ruangan + 1 );
        printf("%d ", hasil_akhir);
    }
return 0;
}